ooni/nettests/third_party/psiphon.py

```diff
@@ -88,8 +88,10 @@ def test_psiphon(self):
         log.debug('PsiphonTest.test_psiphon')
 
         self.report['success'] = None
+		self.report['psiphon_installed'] = None
```

Comment:
I think the indentation of this line is off (looks like some tabs are in here instead of spaces).